data/free worlds war jobs.txt

```diff
@@ -0,0 +1,783 @@
+
+
+
+mission "FW Reinforcements [0]"
+	job
+	repeat
+	name `Reinforcements to <planet>`
+	description `These <bunks> Free Worlds soldiers require immediate transport to <destination> by <date> in order to reinforce the frontlines. Payment is <payment>.`
+	deadline 1 1
+	passengers 5 5 .2
+	to offer
+		random < 20
+		has "salary: Free Worlds"
+		not "event: fw armistice"
+	source
+		government "Free Worlds"
+		not
+			neighbor government "Republic"
+		not
+			neighbor
+				neighbor government "Republic"
+	destination
+		distance 2 10
+		government "Free Worlds"
+		neighbor
+			neighbor government "Republic"
+	on complete
+		payment 10000 180
+		dialog `You unload the Free Worlds soldiers into the busy spaceport of <planet>, and collect your payment of <payment>.`
+
+mission "FW Reinforcements [1]"
+	job
+	repeat
+	name `Reinforcements to <planet>`
+	description `These <bunks> Free Worlds soldiers require immediate transport to <destination> by <date> in order to reinforce the frontlines. Payment is <payment>.`
+	deadline 1 1
+	passengers 5 5 .2
+	to offer
+		random < 10
+		has "salary: Free Worlds"
+		not "event: fw armistice"
+	source
+		government "Free Worlds"
+		not
+			neighbor government "Republic"
+		not
+			neighbor
+				neighbor government "Republic"
+	destination
+		distance 2 10
+		government "Free Worlds"
+		neighbor
+			neighbor government "Republic"
+	on complete
+		payment 10000 180
+		dialog `You unload the Free Worlds soldiers into the busy spaceport of <planet>, and collect your payment of <payment>.`
+
+
+
+mission "FW Outfitter Resupply [0]"
+	job
+	repeat
+	name `Resupply <planet>`
+	description `The outfitter on <planet> is running low on supplies and needs to be restocked immediately. Deliver <cargo> to <destination> by <date>. Payment is <payment>.`
+	deadline 1 1
+	cargo random 5 2 .1
+	to offer
+		random < 50
+		has "salary: Free Worlds"
+		not "main plot completed"
+	source
+		government "Free Worlds"
+	destination
+		distance 4 8
+		government "Free Worlds"
+		attributes outfitter
+	on complete
+		payment 2000 180
+		dialog `You drop off your cargo of <commodity> and collect your payment of <payment>. A Free World captain stationed on <planet> thanks you for delivering the much needed supplies.`
+
+mission "FW Outfitter Resupply [1]"
+	job
+	repeat
+	name `Resupply <planet>`
+	description `The outfitter on <planet> is running low on supplies and needs to be restocked immediately. Deliver <cargo> to <destination> by <date>. Payment is <payment>.`
+	deadline 1 1
+	cargo random 5 2 .1
+	to offer
+		random < 40
+		has "salary: Free Worlds"
+		not "main plot completed"
+	source
+		government "Free Worlds"
+	destination
+		distance 5 10
+		government "Free Worlds"
+		attributes outfitter
+	on complete
+		payment 2000 180
+		dialog `You drop off your cargo of <commodity> and collect your payment of <payment>. A Free World captain stationed on <planet> thanks you for delivering the much needed supplies.`
```

Comment:
`A Free World captain` should be `A Free Worlds captain`, I think. Since "Free Worlds" is a proper name, dropping pluralization doesn't make sense in this case. This also applies to similar missions further down.